zict/tests/test_buffer.py

```diff
@@ -63,6 +63,31 @@ def test_simple():
     assert buff['b'] == 1000
 
 
+def test_setkey_fastslow():
```

Comment:
Nitpick: could you update the name of this test 

```suggestion
def test_setitem_avoid_fast_slow_duplicate():
```